lib/magma/attribute.rb

```diff
@@ -125,40 +154,8 @@ def entry
       end
     end
 
-    def update_option(opt, new_value)
```

Comment:
Thanks to `Sequel::Model` we don't have to write/use things like `Magma::Attribute#update_option` 🎉. This also means we don't have to deal with the `INSERT CONFLICT` weirdness @graft was seeing.